napari/utils/settings/_manager.py

```diff
@@ -119,7 +119,10 @@ def _load(self):
 
         if path.is_file():
             with open(path) as fh:
-                data = safe_load(fh.read()) or {}
+                try:
+                    data = safe_load(fh.read()) or {}
+                except Exception:
+                    data = {}
```

Comment:
Should I catch the specific issue you encountered and raise a Warning or similar @tlambert03 ?